.gitlab-ci-docker-build-multiplatform.yml

```
@@ -0,0 +1,90 @@
+variables:
+  # Required variables
+  DOCKER_REGISTRY:
+    description: "The docker registry for publishing the image (e.g. 'docker.io')"
+  DOCKER_REPOSITORY:
+    description: "The docker registry to publish the image (e.g. 'northerntechhq/northern.tech')"
+  DOCKER_USERNAME:
+    description: "The username for DOCKER_REGISTRY"
+  DOCKER_PASSWORD:
+    description: "The password for DOCKER_REGISTRY"
+
+  # Optional variables
+  BUILDX_DRIVER:
+    description: "The docker buildx builder driver"
+    value: "docker-container"
+  DOCKERFILE:
+    description: "Path to the dockerfile for building relative to DOCKER_DIR"
+    value: "Dockerfile"
+  DOCKER_DIR:
+    description: "The docker build context directory"
+    value: "."
+  MULTIPLATFORM_TARGETS:
+    description: "Comma-separated list of target 'os/architecte' to build"
+    value: "linux/amd64,linux/arm64"
+
+  DOCKER_VERSION:
+    description: "Version of Docker image `docker.io/library/docker` for cli and buildx service"
+    value: "24.0.6"
+  SKOPEO_VERSION:
+    description: "Version of Docker image `quay.io/skopeo/stable` for publishing images"
+    value: "v1.3.1"
+
+
+stages:
+  - build
+  - publish
+
+build:docker:
+  tags:
+    - mender-qa-worker-generic-light
+  stage: build
+  needs: []
+  variables:
+    DOCKER_BUILDKIT: 1
```

Some containers like the `workflows-worker` uses different tag but are starting from the same ${CI_REGISTRY_IMAGE}. By moving the tag definition to the variables section, we could easily override the tag for this kind of images.

```suggestion
    DOCKER_BUILDKIT: 1
    GITLAB_REGISTRY_TAG: '${CI_REGISTRY_IMAGE}:${CI_PIPELINE_ID}'
```

---

Added `CI_IMAGE_REGISTRY_TAG_PREFIX` variable (not sure if built-in variables are available at that scope).